src/components/LicenseText.vue

```diff
@@ -122,7 +119,9 @@ export default {
                 workAttrs.property = 'dct:title'
```

Comment:
And in line 117, only destructure `workTitle` from `this.attributionDetails`, delete `workUrl` from there.

Reply:
Done 👍